mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -171,6 +181,7 @@ def __init__(self,
                  use_sigmoid=False,
                  use_mask=False,
                  reduction='mean',
+                 bce_expand_one_hot=True,
```

Comment:
I have a question.  Wich is better to put this arg in `_init_` or in `forward`.
Can @BIGWangYuDong @jbwang1997 give some advice?
If put in `_init_`, it is convenient to switch the loss in the config file directly, but the disadvantage is that ce loss and mask ce loss do not have this parameter. I don't know if it is suitable or not to put an arg that only belongs to a part of `CrossEntropyLoss` in `_init_`.

Reply:
The original bce loss can directly input onehot labels. When the `pred` and `label` have the same `dim`, the program will skip the `_expand_onehot_labels` function. So I think it's unnecessary to set `bce_expand_one_hot`.

Reply:
> The original bce loss can directly input onehot labels. When the `pred` and `label` have the same `dim`, the program will skip the `_expand_onehot_labels` function. So I think it's unnecessary to set `bce_expand_one_hot`.

It seems that the labels are not one-hot in maskformer.

Reply:
Yes, the label can also be float if it doesn't pass the `bce_expand_one_hot` function.

Reply:
yes, `bce_expand_one_hot` is unnecessary. The docstring is misleading. I have updated it.